app/controllers/documents_controller.rb

```diff
@@ -12,7 +12,7 @@ class DocumentsController < AuthenticationController
   before_action :disable_flash_header, only: :index
 
   def index
-    @documents = policy_scope(@planning_application.documents).order(:created_at)
+    @documents = @planning_application.documents.order(:created_at)
     @planning_application.documents_validated_at ||= @planning_application.created_at
```

Comment:
This line is no longer setting the validated_at date in the documents index

Reply:
Good catch! This was actually prefilling the validation form, which has been moved to it's own page. I have now moved where we did this.